ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/PartyManagementItUtils.scala

```diff
@@ -3,7 +3,13 @@
 
 package com.daml.ledger.api.testtool.suites.v1_8
 
-import com.daml.ledger.api.testtool.infrastructure.participant.ParticipantTestContext
```

Comment:
Probably, not sure if it's worth the effort changing it. I have a bad experience with case-only changes in filenames on Windows

Reply:
You can rename it to PartyManagementITBase ;) maybe it would fit better to a simplified inheritance approach anyway.
Anyway if you have a strong opinion on the name, you can also keep it

Reply:
Renamed as suggested :)